src/components/CtaSection.tsx

```diff
@@ -87,32 +105,39 @@ const Cta: FunctionComponent<Cta> = ({ text, icon, button = false, link, onClick
  * This is our CTA Section as defined in our DLS. Please refer to it for specs.
  *
  * @param props - component props
+ * @param props.background - background for the section
  * @param props.title - a title for the section
  * @param props.description - a description for the section
  * @param props.centerContent - a boolean to center the content
+ * @param props.slimWidth - a boolean for a slim width container
  * @param props.cta1 - cta item 1
  * @param props.cta2 - cta item 2
  */
 export const CtaSection: FunctionComponent<CtaSection> = ({
+    background,
     title = 'Get started with Sourcegraph',
     description = 'Understand, fix, and automate changes across your entire codebase.',
     centerContent = false,
+    slimWidth,
     cta1,
     cta2,
 }) => (
-    <ContentSection color="white" className="">
+    <ContentSection background={background}>
         <div
             className={classNames({
                 'tw-max-w-xl tw-mx-auto tw-text-center': centerContent,
                 'tw-grid tw-grid-cols-5': !centerContent,
+                'tw-grid tw-grid-cols-12 tw-gap-sm tw-grid-flow-row tw-items-center': slimWidth,
```

Comment:
The above suggestion & this should help the text align on the blog page

```suggestion
                'tw-max-w-[846px] tw-mx-auto tw-grid tw-grid-cols-2 tw-items-start': slimWidth,
```
BEFORE:
![image](https://user-images.githubusercontent.com/59381432/187766401-f0e2a74e-9c15-4216-9922-17f70823d0a6.png)
AFTER:
![image](https://user-images.githubusercontent.com/59381432/187766063-a6d6647f-b71a-429b-800d-986f2f2e36b0.png)

Reply:
This doesn't match the DLS spec but it's a similar approach using max width vs grid columns and grid gaps. We want the left content to stay within 4-5 columns. The buttons also need to be vertically centred with the left column content instead of using items-start.